components/src/core/components/Input/CheckboxInput.vue

```diff
@@ -64,6 +69,7 @@ export default defineComponent({
   data(): State {
     return {
       focused: false,
+      checkedProxy: {},
```

Comment:
In the previous implementation, multiple check boxes bound to one v-model will not work as expected.
```
<oxd-checkbox value="one" v-model="test"/>
<oxd-checkbox value="two" v-model="test"/>
```
if we printed `test` when both check boxes are checked, it would be like `test : 'one'` or `test : 'two'` depending on order we check the check boxes. Ideally it should be like `test: [one, two]`
This implementation was referred from [vue documentation](https://v3.vuejs.org/guide/component-basics.html#using-v-model-on-components)

PS: This is favourable since we plan to use this check boxes in oxd-table as well.